homeassistant/helpers/translation.py

```diff
@@ -197,23 +197,25 @@ def __init__(self, hass: HomeAssistant) -> None:
         """Initialize the cache."""
         self.hass = hass
         self.loaded: dict[str, set[str]] = {}
-        self.cache: dict[str, dict[str, dict[str, Any]]] = {}
+        self.cache: dict[str, dict[str, dict[str, dict[str, str]]]] = {}
```

Comment:
There’s no actual change here in the format of the cash, except the typing was missing for the last level